src/main/scala/nl.knaw.dans.easy.validatebag/rules/metadata/package.scala

```diff
@@ -296,18 +300,19 @@ package object metadata extends DebugEnhancedLogging {
     }
   }
 
-  // Please note that this method does accept all elements within 'files \ file' that are in the
-  // allowed namespaces, not only those that are valid! This is, however, not a problem, since
-  // invalid elements will be caught by `filesXmlConformsToSchemaIfDeclaredInDefaultNamespace`
   def filesXmlFilesHaveOnlyAllowedNamespaces(t: TargetBag): Try[Unit] = {
     trace(())
     t.tryFilesXml.map { xml =>
-      val fileChildren = xml \ "file" \ "_"
-      val hasOnlyDcTermsInFileElements = fileChildren.forall {
-        case n: Elem => allowedFilesXmlNamespaces contains xml.getNamespace(n.prefix)
-        case _ => true // Don't check non-element nodes
+      if (xml.namespace == filesXmlNamespace) {
+        debug("Rule filesXmlFilesHaveOnlyAllowedNamespaces has been checked by files.xsd")
+      } else {
+        val fileChildren = xml \ "file" \ "_"
+        val hasOnlyDcTermsInFileElements = fileChildren.forall {
```

Comment:
I know, it's my mistake (forgot it in #28), but could you change `hasOnlyDcTermsInFileElements` to `hasOnlyAllowedNamespacesInFileElements`?

Reply:
👍